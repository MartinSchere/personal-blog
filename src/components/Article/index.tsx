import React from 'react';
import "./styles.scss";

type ArticleProps = {

}

const Article = (props:ArticleProps) => {
return (
<div>
  Article
</div>
);
};

export default Article;