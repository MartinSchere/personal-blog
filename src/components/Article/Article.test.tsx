import React from 'react';
import Article from './index';
import renderer from 'react-test-renderer';

describe('Article', () => {
it('renders without error', () => {
const tree = renderer
.create(<Article></Article>)
.toJSON();
expect(tree).toMatchSnapshot();
});
});