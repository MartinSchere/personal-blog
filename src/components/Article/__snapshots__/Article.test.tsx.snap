// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article renders without error 1`] = `
<div>
  Article
</div>
`;
